src/sentry/api/serializers/models/incident.py

```diff
@@ -82,7 +89,7 @@ def serialize(self, obj, attrs, user):
         context["isSubscribed"] = attrs["is_subscribed"]
         context["seenBy"] = seen_list["seen_by"]
         context["hasSeen"] = seen_list["has_seen"]
-        context["alertRule"] = serialize(obj.alert_rule, user)
```

Comment:
No need to include this here, since this is already included when we call `super`